docs/includes/_access_token_create_conf_client_creds.mdx

```diff
@@ -1,55 +1,55 @@
 
 <AddAppAdminConsole />
 
-3. On the **External Protocol** tab, use the following values to complete this tab.  
-
-  | Property | Value | 
-  |---|---|  
-  |**Protocol**|OAuth2|  
-  |**Client Type**|Confidential|
-  |**PKCE**|Disabled|
-  |**Token Endpoint Auth Method**|Client Secret Basic|
-  |**Grant Type**|Client Credentials|  
-
-4. Click **Submit** to save the new app.  
-
-5. Next, create the HTTP request:  
-
-      **Request url:**  
-    
-      ```http
-      https://auth-{us|eu}.beyondidentity.com/v1/tenants/{tenant_id}/realms/{realm_id}/applications/{application_id}/token
-      ```
-    
-      **Request method:** POST  
-    
-      **Request headers:**  
-      
-      ```http
-      Authorization: Basic {client_credentials}
-      content-type: application/x-www-form-urlencoded  
-      ```
-    
-      where {client_credentials} is value of the app's **Client ID** and **Client Secret**, concatenated together in the format '{client_id}:{client_secret}' and base64 encoded  
-    
-      **Request body:**  
-    
-      ```http
-      grant_type: "client_credentials"
-      scope: {scopes}
-      expiration_time: {seconds}
-      custom_claims: {custom_claims_json}
-      ```
-    
-      where:  
-
-       - {scopes} is one or more of the app's **Allowed Scopes**, space delimited.  
-
-       - expiration_time is an OPTIONAL parameter, and {seconds} is the desired time after minting, in seconds, for which the token will be considered valid. If included, this value must be less than or equal to the app's configured **Expires** setting  
-
-       - custom_claims is an OPTIONAL parameter, and {custom_claims_json} is a JSON string containing the desired additional claims as key value pairs, for example '{"a": "b", "c": "d"}'. Upon successful token request, the resulting token will contain additional field 'bi_custom' with the desired claims.
+3. On the **External Protocol** tab, use the following values to complete this tab.
+
+  | Property | Value |
+  | --- | --- |
+  | **Protocol** | OAuth2 |
+  | **Client Type** | Confidential |
+  | **PKCE** | Disabled |
+  | **Token Endpoint Auth Method** | Client Secret Basic |
+  | **Grant Type** | Client Credentials |
+
+4. Click **Submit** to save the new app.
+
+5. Next, create the HTTP request:
+
+  **Request url:**
+
+  ```http
+  https://auth-{us|eu}.beyondidentity.com/v1/tenants/{tenant_id}/realms/{realm_id}/applications/{application_id}/token
+  ```
+
+  **Request method:** POST
+
+  **Request headers:**
+
+  ```http
+  Authorization: Basic {client_credentials}
+  content-type: application/x-www-form-urlencoded
+  ```
+
+  where {client_credentials} is value of the app's **Client ID** and **Client Secret**, concatenated together in the format '{client_id}:{client_secret}' and base64 encoded
+
+  **Request body:**
+
+  ```http
+  grant_type: "client_credentials"
+  scope: {scopes}
+  expiration_time: {seconds}
+  custom_claims: {custom_claims_json}
+  ```
```

Comment:
It might be helpful to use the `MultiLanguageCodeBlock` and generate a curl here too